subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -29,7 +29,7 @@ struct lll_conn_iso_stream {
 
 	/* Connection parameters */
 	uint8_t  access_addr[4];    /* Access address */
-	uint32_t offset;            /* Offset of CIS from start of CIG in us */
+	uint32_t ticks_offset;      /* Offset of CIS from start of CIG */
```

Comment:
why LLL uses coarse tick units?

(The goal is, use of fine packet timing units of 1 us inside LLL. But, yes there is an exception that scanner is using ticker to set scanner window stop, other than this, can we try using ticker and its units in ULL only?)

Reply:
Yes, this is an oversight on my behalf. The reason is that we have sub-microsecond tick units, so this is the optimal implementation for us. Will change to use us resolution for consistency.